R/g.part5.R

```diff
@@ -481,9 +481,10 @@ g.part5 = function(datadir = c(), metadatadir = c(), f0=c(), f1=c(),
                             }
                             if (timewindowi == "MM" & si > 1) { # because first segment is always full window
                               if (("segment" %in% colnames(ts)) == FALSE) ts$segment = NA
-                              ts$segment[segStart:segEnd] = si
+                              if (!is.na(segStart) & !is.na(segEnd)) {
```

Comment:
Please use `&&` here because you are comparing single values and not a vector. I am now trying to use `&&` and `&` distinction more systematically when I edit GGIR.